### DISCUSSION TITLE
Teleport 시 캐릭터가 땅에 뚫립니다

### DISCUSSION THREAD

**Discussion by JeongYire** [naverz/zepeto-studio-kor#371] [5. Q&A]
Teleport 기능을 만드려고 ZepetoCharacter의 Teleport 기능을 사용하였습니다.
이 경우 정상적으로 위치가 이동이 되나 잠깐동안이지만 캐릭터가 땅을 인지하지못하고 땅에 몸이 뚫리는 순간이 생깁니다. 
그래서 위치값을 좀 높게 잡아서 공중에서 시작하게했으나 똑같은 현상이 발생합니다.

이 경우 어떻게 해야하는지요?

**Comment by ted002** [comment]
비슷한 현상이 저한테도 있어서 몇가지 말씀을 같이 드리자면 제페토 자체의 텔레포트 애니메이션의 중앙이 밑으로 내려가서 생기는 일 같습니다. 콜라이더는 그대로인데 애니메이션이 이상해서 그런것 같습니다

**Reply by JeongYire** [reply]
저도 확인해보니 콜라이더는 그대로인데 가운데지점이 밑으로 내려가네요...

**Comment by moondory77-naverz** [comment]
안녕하세요. 
초기상태의 ZepetoAnimator 컴포넌트에서, Teleport State의 clip은 빈 슬롯(None) 상태로 제공하기 때문에 그렇습니다.
이 경우, Teleport State에 임의의 클립을 적용하면 해당 현상이 발생하지 않을 듯 합니다.

다음 업데이트에, 초기상태 Animator의 TeleportState에 대해서도 디폴트 클립을 적용하도록 하겠습니다.
감사합니다.

**Reply by smilejsu82** [reply]
월드 버전 1.1.5에서 수정되었나요? 아직 뚫리는거 같은데요
